deltachat-jsonrpc/src/api/types/calls.rs

```diff
@@ -51,3 +52,82 @@ impl JsonrpcCallInfo {
         })
     }
 }
+
+#[derive(Serialize, TypeDef, schemars::JsonSchema)]
+#[serde(rename = "CallState", tag = "kind")]
+pub enum JsonrpcCallState {
+    /// Fresh incoming call that should be still ringing.
+    IncomingRinging,
```

Comment:
for the states that are valid for both, incoming and outgoing, this only makes the API more difficult to use, one will anyways first check if the call is incoming/outgoing via the msg object or the call_info api because that is needed information anyways to know how to display the bubbles etc. and then one have to take care to use the proper prefix depending on incoming/outgoing, while both share the same text and there is no need for the distinction